components/blocks/Landing/LandingLeaderboards.vue

```diff
@@ -14,8 +14,11 @@ defineProps<LandingLeaderboardsProps>()
     id="landing-leaderboards"
     class="mb-4 grid w-full grid-cols-1 content-start items-start gap-4 lg:grid-cols-2"
   >
+    <!-- TODO: Get design for no data edge case. Also get this whole page's new design. -->
```

Comment:
This should never happen. If the leaderboards can't be fetched, have the parent component display an error like "error fetching leaderboards" instead of this component.